### ISSUE TITLE
PyQt5: Could not load requested Qt binding.

### ISSUE THREAD

**Issue by dc740** [ipython/ipython#8411]
I'm running python 3 in ubuntu 15.04, and latest ipython release upgraded from pip, and pyqt5 was not detected correctly.
Fixing the missing QtSvg module took me 30 seconds (apt-get install python3-pyqt5.qtsvg), and finding out why iPython would refuse to load PyQt5 took me an hour of reinstalling libraries until I opened the source code.

A descriptive error message would be "nice"

Here is the actual error message:
    Currently-imported Qt library:   'pyqt5'
    PyQt4 installed:                 False
    PyQt5 installed:                 False
    PySide >= 1.0.3 installed:       False
    Tried to load:                   ['pyqt5']

After a lot of debugging, and even installing in virtualenv... I found out that "has_binding" only returns a boolean value inside "IPython/external/qt_loaders.py", so if you have a missing package like QtSvg (which was my case in my regular installation and also in my virtualenv)... then it will return false, even when PyQT5 is actually installed.

**Comment by jgehrcke** [comment]
Same for me:
```
Currently-imported Qt library: 'pyqt5'
PyQt4 installed: False
PyQt5 installed: False
PySide >= 1.0.3 installed: False
Tried to load: ['pyqt5']
```
This is a highly confusing error message, leading to ridiculous debugging work, whereas a proper traceback or error message would have shown immediately that it is just QtSvg which is missing.

Others are also suffering from this confusing error message: http://askubuntu.com/questions/704534/python3-ipython-pylab-on-ubuntu-14-04

**Comment by Carreau** [comment]
could that be considered fixed by #9104 ?

**Comment by takluyver** [comment]
Yep, I think it's significantly clearer now.
